src/python_testing/matter_testing_infrastructure/chip/testing/matter_testing.py

```diff
@@ -612,6 +612,142 @@ def test_skipped(self, filename: str, name: str):
         logging.info(f"Skipping test from {filename}: {name}")
 
 
+class PIXITType(Enum):
```

Comment:
I know I used PIXIT in the issue, but I think this applies generally to test flags. But I can't think of a great name for just "Flags" that doesn't sound too generic. Ideas welcome.

Reply:
To be honest, I also don't have many ideas. TestFlag? TestFlagType? TestParamType?